<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMultiUpload</class>
 <widget class="QDialog" name="DialogMultiUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备文件上传</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设备列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>全部上传</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>本地文件路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtLocalFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart_2">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>远程文件路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtTarget">
       <property name="text">
        <string>/../etc/hotplug.d/block/10-mount</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
